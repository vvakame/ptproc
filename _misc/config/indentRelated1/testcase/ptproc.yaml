mapfile:
  disableRewriteIndent: true
maprange:
  disableDedent: true
  disableRewriteIndent: true
