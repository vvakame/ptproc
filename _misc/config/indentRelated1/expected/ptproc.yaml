mapfile:
  startRegExp: "mapfile:([^\\s]+)"
  endRegExp: mapfile.end
  disableRewriteIndent: true
  indentWidth: 2
  defaultSkip: 0
maprange:
  startRegExp: "maprange:([^\\s]+)"
  endRegExp: maprange.end
  disableDedent: true
  disableRewriteIndent: true
  indentWidth: 2
  defaultSkip: 0
